perception/image_projection_based_fusion/CMakeLists.txt

```diff
@@ -1,12 +1,19 @@
 cmake_minimum_required(VERSION 3.14)
 project(image_projection_based_fusion)
-add_compile_options(-Wno-unknown-pragmas)
+# add_compile_options(-Wno-unknown-pragmas)
+add_compile_options(-Wno-unknown-pragmas -fopenmp)
```

Comment:
Cloud you add `fopenmp` to https://github.com/tier4/autoware-spell-check-dict ?

Reply:
rerun spell-check
https://github.com/autowarefoundation/autoware.universe/actions/runs/5618329541?pr=4314